components/datadog/agent/docker_image.go

```diff
@@ -17,7 +17,20 @@ const (
 	defaultOTAgentImageTag       = "nightly-ot-beta-main"
 )
 
-func dockerAgentFullImagePath(e config.Env, repositoryPath, imageTag string, otel bool) string {
+func dockerAgentFullImagePath(e config.Env, repositoryPath, imageTag string, otel bool, useLatestStableAgent bool) string {
+	if repositoryPath == "" && otel {
+		repositoryPath = defaultOTAgentImageRepo
+	}
+
+	if repositoryPath == "" {
+		repositoryPath = defaultAgentImageRepo
+	}
+
+	// return latest stable agent image path
+	if useLatestStableAgent {
+		return utils.BuildDockerImagePath(repositoryPath, "latest")
+	}
```

Comment:
I think we should not need that extra parameters. When we want `latest` we can just pass `imageTag= latest`